### PR TITLE
bump aws provider

### LINKED ISSUE #37: bump aws provider to latest stable (3.74.0)
resolve https://github.com/aeternity/terraform-aws-api-gateway/issues/37

### PR DESCRIPTION
**Pull request by michalbagrowski**
resolve https://github.com/aeternity/terraform-aws-api-gateway/issues/37

### CODE REVIEW

File: test/test.tf [hunk 1]
```diff
@@ -63,7 +63,7 @@ module "test_nodes_sydney_channels" {
 }
 
 module "test_lb_sydney" {
-  source                    = "github.com/aeternity/terraform-aws-api-loadbalancer?ref=master"
+  source                    = "github.com/aeternity/terraform-aws-api-loadbalancer?ref=bump-aws-provider"
```
Comment (michalbagrowski) [inline test/test.tf]: this require merge of: https://github.com/aeternity/terraform-aws-api-loadbalancer/pull/14 and will ba set back to `master`

**Review by mmpetarpeshev — DISMISSED** [review]
Not aware of all versions , otherwise it LGTM.

**Review by venimus — APPROVED** [review]

**Comment by dincho** [comment]
```
╷
│ Warning: Provider aws is undefined
│ 
│   on test.tf line 31, in module "test_nodes_sydney":
│   31:     aws = aws.ap-southeast-2
│ 
│ Module module.test_nodes_sydney does not declare a provider named aws.
│ If you wish to specify a provider configuration for the module, add an entry for aws in the required_providers block within the module.
│ 
│ (and one more similar warning elsewhere)
```
This warning concerns me a bit, is it easy to fix ?

**Comment by michalbagrowski** [comment]
@dincho already fixed (https://github.com/aeternity/terraform-aws-aenode-deploy/pull/34/), i have rerun test no warning

**Review by dincho — APPROVED** [review]

**Pull request merged**